### PR TITLE
Update rest.py

### PR DESCRIPTION
**Pull request by TusharSharma-Dassana**
Made some changes to handle DateTime error

### CODE REVIEW

File: dassana/rest.py [hunk 1]
```diff
@@ -8,6 +9,11 @@
 from google.cloud import pubsub_v1
 
 
+logging.basicConfig(level=logging.INFO)
```
Comment (vinay-dassana) [inline dassana/rest.py:12]: not required

File: dassana/rest.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import requests
 import gzip
+import logging
```
Comment (vinay-dassana) [inline dassana/rest.py:3]: not required

File: dassana/rest.py [hunk 3]
```diff
@@ -62,16 +68,19 @@ def forward_logs(
 
     if bytes_so_far > 0:
         payload_compressed = gzip.compress(payload.encode("utf-8"))
-        response = requests.post(
+        response = http.post(
```
Comment (vinay-dassana) [inline dassana/rest.py:71]: revert this

File: dassana/rest.py (hunk 1 above)
Comment (TusharSharma-Dassana) [reply dassana/rest.py]: done

File: dassana/rest.py (hunk 2 above)
Comment (TusharSharma-Dassana) [reply dassana/rest.py]: done

File: dassana/rest.py (hunk 3 above)
Comment (TusharSharma-Dassana) [reply dassana/rest.py]: done

**Review by yatin-dassana — APPROVED** [review]

**Pull request merged**